### DISCUSSION TITLE
Question on chosing Hyperparameter

### DISCUSSION THREAD

**Discussion by zjucm** [brucefan1983/GPUMD#784] [Q&A] [answered]
Dear author:
I am wondering how can I choose the appropriate hyperparameter based on my own dataset？For example , I have a SiC dataset. Are there any stratgies？

**Comment by brucefan1983** [comment]
Usually it is a good practice to start from the default values (see the bottom of this page: https://gpumd.org/nep/input_files/nep_in.html). For SiC, I would like to change the cutoff to `5 4` and use default values for all other hyperparameters. So your `nep.in` would be
```
type 2 C Si
cutoff 5 4 # you can also try (5 5), or (6 4), or (6 5)
```
After this, you can try to modify one hyperparameter at one time, and observe how the accuracy (preferably for a testset) and speed change.

**Reply by zjucm** [reply]
Thank you！I would also like to learn if you have any suggestions on n_max or basis_size ，if there is，I would be more than appreciated!

**Comment by brucefan1983** [comment, marked as answer]
For `basis_size`, jsut use the default values, for `n_max`, larger values give higher accuracy but lower speed. The defaut values are `4 4`, which give relatively low accuracy but high speed. You can try to increase `n_max` to `6 6` or `8 6`. Do it and check results, you will gain experiences quickly.

**Reply by zjucm** [reply]
OK!Thank you!
